src/views/partials/user-menu.js

```diff
@@ -0,0 +1,57 @@
+import { getMessage } from '../../utils/fluent.js'
+
+const userMenuButton = data => `
+<button
+  aria-expanded='false'
+  aria-haspopup='true'
+  class='user-menu-button'
+  title='${getMessage('menu-button-title')}'
+>
+  <img src='${data.avatar}' alt='${getMessage('menu-button-alt')}' />
+</button>
+`
+
+const userMenuPopover = data => `
+  <ul class='user-menu-list user-menu-popover' role='menu' hidden>
```

Comment:
That’s a good point. I’m not sure if it would be semantically correct to nest a `<menu>` inside a `<menu>`, but in that case I probably should also drop the `role` attribute.